packages/compiler/core/program.ts

```diff
@@ -1,10 +1,17 @@
 import { createBinder } from "./binder.js";
 import { Checker, createChecker } from "./checker.js";
 import { createSourceFile } from "./diagnostics.js";
-import { SymbolFlags } from "./index.js";
+import { compilerAssert, MANIFEST, SymbolFlags } from "./index.js";
+import { LIBRARIES_URL_LOADED } from "./library.js";
```

Comment:
nit: can we export this as a method (e.g. `getLoadedLibraries()`) instead of directly exporting the set object? It's fine if the method just returns the set, but it feels weird to reference an ALL_CAPS_EXPORT